dali/kernels/erase/erase_gpu.h

```diff
@@ -376,10 +378,9 @@ struct EraseGpu {
   using sample_t = erase_sample_desc<T, ndim>;
 
   KernelRequirements Setup(KernelContext &context,
-                          //  const TensorListView<StorageGPU, const T, ndim> &in
                            const InListGPU<T, ndim> &in,
-                           const InListGPU<ibox<ndim>, 1> &erased_regions,
-                           span<const T> fill_values = {}) {
+                           const std::vector<std::vector<ibox<ndim>>> &erased_regions,
```

Comment:
Not really a fan of vector<vector<>>, especially as this data can be easily packed as one, contiguous allocation.

Reply:
There should be a possibility to specify the regions directly in GPU memory - to make it possible to use GPU Shapes operator, for example.